### PR TITLE
Support added for rendering markdown files as templates

### LINKED ISSUE #29: Add support for markdown files as templates
I use markdown as my main markup language for notes and other documents. It'd awesome if you could add a support for markdown files, I'd like to use my markdown files as html pages. Maybe a simple converter like [PythonMarkdown](https://github.com/Python-Markdown/markdown)'s might help(see reference), thanks in advance!

reference: https://python-markdown.github.io/reference/#output_format

### LINKED ISSUE #31: Add code highlight in templates
When code is present in a template there should be a way to highlight it according to the programming language it was written

### PR DESCRIPTION
**Pull request by Sddilora**
closes #29
closes #31

### CODE REVIEW

File: templates/index.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+```
+
+```js
+console.log('hello world')
+var str = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+```
+
+[waa](www.google.com)
+
+> sth
```
Comment (log101) [inline templates/index.md]: I suppose these lines should be removed.

**Comment by log101** [comment]
@Sddilora Does syntax highlighting depends on a single static css file? If so would it work on different languages accurately? Is there a way highlight it automatically using pygment library without depending on a static css file?

File: LumosWeb/md_to_html.html
Comment (log101) [inline LumosWeb/md_to_html.html:1]: IMHO the syntax highlighting should work automatically without depending a static css.

**Review by log101 — CHANGES_REQUESTED** [review]
Check out the comments

**Comment by Sddilora** [comment]
> Check out the comments
@log101 
I have made some changes, but the Pygments CodeHilite feature does not perform syntax highlighting without CSS. To ensure proper functionality and prevent any potential issues in the future, I have separated the CSS file.

**Review by log101 — APPROVED** [review]
Great work, I'll try to use it to serve my personal blog next week! 🎉

**Pull request merged**